### DISCUSSION TITLE
Default testing device

### DISCUSSION THREAD

**Discussion by borrmann** [dotnet/maui#10617] [Ideas]
### Description

Since there are still many issues open it is really hard to develop Apps and especially migrating existing ones from Xamarin.Forms to .NET MAUI. 
Life could potentially be a bit easier, if there was a standard device which could be used for development and pre-stable versions, so we would know the expected behaviour once bugs are fixed (for the others). 
My app looks entirely different on Android an iOS (Colours, Margins, WordWrapping etc. etc.) and I don't know which version I should try to make look how I want it. If I change things on one device, it often doesnt behave the same way on another. If I knew there was a device you guys are mainly developing with (or for) first and then the code for other devices are derived from it, this could really help. Maybe at least per platform this would be really helpful!  

### Public API Changes

None

### Intended Use-Case

In my App I don't know for which devices I should develop for since I am expecting large changes of the Framework in many areas.

**Comment by PureWeen** [comment]
@davidortinau
